molgenis-data-migrate/src/main/java/org/molgenis/data/version/MolgenisVersionService.java

```diff
@@ -39,7 +39,7 @@
 @Service
 public class MolgenisVersionService
 {
-	public static final int CURRENT_VERSION = 13;
+	public static final int CURRENT_VERSION = 12;
```

Comment:
Are we absolutely sure nobody will be using 1.8.0? Decreasing the version here will cause migration problems in the future for these users.